docs/kepler_model_server/pipeline.md

```diff
@@ -33,20 +24,7 @@ Energy/power source|Energy/power components
 [acpi](../design/kepler-energy-sources.md#using-kernel-driver-xgene-hwmon)|platform
 ||
 
-## Idle power/Control plane power
-
-`isolate` is a mechanism to separate the power portion that is consumed on the node in an idle state or the power portion that is consumed by operating systems and control plane processes. These portions of power is more than zero even if the metric utilization of workload is zero. We called the models those are trained after isolating these power portions as `DynPower` models. At the same time, the models those are trained without isolation are called `AbsPower` models. `DynPower` model is used to estimate `container power` and `process power` while `AbsPower` model is used to estimate `node power`.
-
-There are two common available `isolators`: *ProfileIsolator* and *MinIdleIsolator*. 
-
-*ProfileIsolator* relies on profiled background powers (profiles) and removes resource usages by system processes from the training while *MinIdleIsolator* assumes minimum power as an idle power and includes resource usages by system processes in the training. 
-
-The pipeline with *ProfileIsolator* will be applied first if the profile that matches the training `node_type` is available. Otherwise, the other pipeline will be applied. 
-
-(check how profiles are generated [here](./node_profile.md))
-
-
-## Available metrics
+## Feature group
 
 `feature group` is an abstract that groups available features based on origin of the resource utilization metrics. On some node environments, some origin can be inaccessible such as hardware counter on the virtual private cloud. The model are trained for each defined group as below.
 
```

Comment:
`feature group` is an abstraction of the available features based on the infrastructure context since some environments might not expose some metrics. For example, on the virtual machine on private cloud, hardware counter metrics are typically not available. Therefore, the models are trained for each defined resource utilization metric group as below.